source/hic_hal/stm32/stm32h743ii/extended_features/include/aux_task.h

```diff
@@ -0,0 +1,14 @@
+#ifndef AUX_TASK_H_
+#define AUX_TASK_H_
+
+#include "cmsis_os2.h"
+
+#define AUX_TASK_STACK (800)
+#define AUX_TASK_PRIORITY (osPriorityNormal)
+#define FLAGS_AUX_100MS         (1 << 0)
```

Comment:
nit (small thing): the space after the macro's name is inconsistent. Generally the all macro values should be vertically aligned by using tabs.